src/python/serenity_types/pricing/derivatives/options/valuation.py

```diff
@@ -138,12 +140,12 @@ class OptionValuationRequest(CamelModel):
     How to derive the forward price for the options: from curves or futures, or disregard the forward.
     """
 
-    vol_surface_id: Optional[UUID]
+    vol_surface_id: Optional[Dict[UUID, UUID]]
     """
     The optional unique ID of the surface to load, latest version as-of the as_of_time.
     """
 
-    vol_surface: Optional[InterpolatedVolatilitySurface]
+    vol_surface: Optional[Dict[UUID, InterpolatedVolatilitySurface]]
     """
     The optional client-provided volatility surface to use. If the client provides neither a VS ID
     nor their own volatility surface, the system will load the default for the underlying as-of the as_of_time.
```

Comment:
Since we are able to specify multiple options with different underlying asset, we should also support multiple vol_surfaces just like yield_curves?

Reply:
If we are going to do this, rename the field to "vol_surfaces" please. And note this is not *must have* for this release: if it becomes an issue to support multiple underliers, just add a validation error.